README.md

```diff
@@ -1,19 +1,29 @@
-[![Build Status](https://sonic-jenkins.westus2.cloudapp.azure.com/job/common/job/linux-kernel-build/badge/icon)](https://sonic-jenkins.westus2.cloudapp.azure.com/job/common/job/linux-kernel-build/)
+[![Build Status](https://dev.azure.com/mssonic/build/_apis/build/status%2FAzure.sonic-linux-kernel?branchName=master)](https://dev.azure.com/mssonic/build/_build/latest?definitionId=13&branchName=master)
 
 # SONiC - Kernel
 
 ## Description
-This repository contains the scripts and patches to build the kernel for SONiC. SONiC uses the same kernel for all platforms. We prefer to out-of-tree kernel platform modules. We accept kernel patches on following conditions:
+This repository contains the scripts and patches to build the kernel for SONiC. SONiC uses the same kernel for all amd64 platforms (armhf and arm64 platforms currently have platform-specific kernels). We accept kernel patches on following conditions:
```

Comment:
I’d add one dot/period before after *platforms* before starting the ()., and move the dot/period at the end inside the (), as it’s a full sentence.